api/rest/serializers.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import permissions, routers, serializers, viewsets
+
+from ..models import Year
+
+
+class YearSerializer(serializers.HyperlinkedModelSerializer):
+    startDate = serializers.DateField(source='start_date')
+    endDate = serializers.DateField(source='end_date')
+    startSignupsAt = serializers.DateField(source='start_date_of_signups')
```

Comment:
Just noticed this and missed it in the model. We need startSignupsAt to be DateTime to be able to say: "You can sign up from 12:00"